packages/core/src/internal-components/panel/panel.element.scss

```diff
@@ -0,0 +1,34 @@
+@import './../../styles/tokens/generated/index';
+
+:host {
+  --width: 100%;
+  --height: 100%;
+  --color: initial;
+  --overflow: visible; // default html behaviour
+  --overflow-x: visible; // default html behaviour
+  --overflow-y: visible; // default html behaviour
+  --background: #{$cds-alias-object-container-background};
+  --border-color: initial;
+  --border-width: #{$cds-alias-object-border-width-100};
+  --border-radius: #{$cds-alias-object-border-radius-100};
+  --padding-horizontal: #{$cds-global-space-0};
```

Comment:
Instead of doing things like `padding-horizontal` and `padding-vertical` we use `--padding` where we can. This is to reduce the scope of the API.